envs/jruby/README.md

```diff
@@ -0,0 +1,3 @@
+# JRuby
+
+Environment to build and run [JRuby](https://www.jruby.org/).
```

Comment:
Build as in build from source?

Reply:
Yea the terminology could be better :)
More like "Environment to execute ruby scripts using the JRuby interpreter"

Reply:
Fixed in 6329cf142f59e258c16a4647d7a53864221637f1

Reply:
Thank you @Mic92 for missing block.